source/apps/rollback.txt

```diff
@@ -1,4 +1,4 @@
-.. _roll-back-deployments:
+.. _rollback-deployments:
```

Comment:
maybe keep ref in case linked to from somewhere downstream

Reply:
I checked there's no downstream refs